### PR TITLE
Add custom dropdown

### LINKED ISSUE #9227: [Docs] Enhance dropdown menu that specifies docs version
### Current State
At present, there is an issue where the primary version button and its associated dropdown menu appear to be conflicting with each other, as depicted in the provided screenshot
Screenshot 2023-10-28 131512
Screenshot 2023-10-28 131526

### Desired State

Add adequate spacing between primary version button and its associated dropdown menu  -

Consider this as a reference -
Screenshot 2023-10-28 131404

---

### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)
- 📚 [Instructions for contributing to documentation](https://docs.meshery.io/project/contributing/contributing-docs)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/contributing/build-and-release)
- 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](http://discuss.meshery.io) and [Community Slack](https://slack.meshery.io)

### PR DESCRIPTION
**Pull request by 11happy**
**Notes for Reviewers**

This PR fixes #9227, Currenlt I have added a custom drop down please see the attached image,

Screenshot from 2024-04-26 06-38-36

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 74f9778fbcacfcb0d2dd2aec7cf0c4be9b00806f at: https://662b014acd3a97456dc8caf3--meshery-docs-preview.netlify.app

File: docs/_sass/navbar.scss [hunk 1]
```diff
@@ -292,4 +292,60 @@ header {
     }
   }
 }
+ .select-dropdown {
+    display: none;
+    position: absolute;
+    top: calc(75% + 5px);
+    width: 83px;
```
Comment (betelgeuse-code) [inline docs/_sass/navbar.scss:299]: We should use `rem`

**Review by iArchitSharma — CHANGES_REQUESTED** [review]
There are couple of problem

1. why is text color and arrow color different?
2. why are you displaying bullets?
3. test font should be same as rest of the header

**Comment by vishalvivekm** [comment]
@11happy 
Adding the item to agenda for website call on Monday: https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit

**Comment by l5io** [comment]
🚀 Preview for commit 6834f2a8033cc90b6ea5cc896cebf4d17a35bb15 at: https://6635af6c240d769649e97c76--meshery-docs-preview.netlify.app

**Comment by 11happy** [comment]
As soon as my academic commitments ease up by 8-9 may, I'll promptly update the PR. Thank you for being patient with this PR.

**Comment by 11happy** [comment]
I have addressed all the above suggestions.

**Comment by l5io** [comment]
🚀 Preview for commit b4a9935b29ea9ddee0dd53d9027e66783ae7d419 at: https://663ce2c9db4c2d00ac8d10fa--meshery-docs-preview.netlify.app

**Comment by vishalvivekm** [comment]
Hey @11happy , let's discuss this on today's website meeting today at 7 AM CT. I am adding this as an [agenda item](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#heading=h.69qskjv558bk). Please Join [https://meet.layer5.io/websites](https://www.google.com/url?q=http://meet.layer5.io/websites&sa=D&source=calendar&usd=2&usg=AOvVaw1CNpNwmYd_rTFcQa_7fBB5) if you are available during the meet.

**Comment by vishalvivekm** [comment]
> Hey @11happy , let's discuss this on today's website meeting today at 7 AM CT. I am adding this as an [agenda item](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#heading=h.69qskjv558bk). Please Join [https://meet.layer5.io/websites](https://www.google.com/url?q=http://meet.layer5.io/websites&sa=D&source=calendar&usd=2&usg=AOvVaw1CNpNwmYd_rTFcQa_7fBB5) if you are available during the meet.

Will you join next websites call on Monday to discuss this item @11happy ?

**Comment by 11happy** [comment]
Yes I will be able to join this monday, I am sorry for not being able to attend last week due to my health issues.

**Comment by 11happy** [comment]
If its okay can we discuss this on dev meet wednesday as today time didn't allow :(

**Comment by iArchitSharma** [comment]
@11happy please remove the border color
image

**Pull request closed without merging**